invisible_cities/reco/pmaps_functions_test.py

```diff
@@ -110,20 +118,3 @@ def test_df_to_pmaps_dict_events_data_correct(s12_dataframe_converted):
 
     assert (converted_time == original.time).all()
     assert (converted_ene  == original.ene ).all()
-
-
-def test_df_to_pmaps_dict_duck_types(KrMC_pmaps):
-    (s1s, s2s, _), _ = KrMC_pmaps
-    s1dict = df_to_pmaps_dict(s1s)
-    s2dict = df_to_pmaps_dict(s2s)
-
-    for d in [s1dict, s2dict]:
```

Comment:
That's from the first commit. The second one removes that test (`test_df_to_pmaps_dict_duck_types`) because the code fits better inside `test_df_to_pmaps_dict_structure`